R/constants.R

```diff
@@ -130,9 +130,11 @@ ELECTRICITY_INPUT_FUELS<- c( "biomass", "coal", "gas", "refined liquids" )
 # and should contain all historical years used by other modules
 socioeconomics.MADDISON_HISTORICAL_YEARS <- seq(1700, 1900, 50) # Years for which to use Maddison data
 socioeconomics.UN_HISTORICAL_YEARS <- c(1950, 1971:2010) # Years for which to use UN data
```

Comment:
I thought something like that might be the case.  All of this makes me wonder if we are really quite certain what is supposed to happen when we tinker with the historical/future years.